tests/i18n/commands/templates/template_0_with_no_error.tpl

```diff
@@ -0,0 +1,6 @@
+{% load i18n %}
+{% blocktranslate %}Hello{% endblocktranslate %}
+
+This file has a name that should be lexiographically before
```

Comment:
Minor typo: "lexicographically"

Thanks for adding a test!

Reply:
Fixed!